model/registry/geo.yaml

```diff
@@ -0,0 +1,113 @@
+groups:
+  - id: registry.geo
+    type: attribute_group
+    prefix: geo
+    brief: >
+      Geo fields can carry data about a specific location related to an event.
+      This geolocation information can be derived from techniques such as Geo IP, or be user-supplied.
+    attributes:
+      - id: city.name
+        stability: experimental
+        type: string
+        brief: >
+          City name.
+        examples: [ 'Montreal', 'Berlin' ]
+      - id: continent.code
+        stability: experimental
+        brief: >
+          Two-letter code representing continent’s name.
+        type:
+          members:
+            - id: 'af'
+              stability: experimental
+              value: 'AF'
+              brief: 'Africa'
+            - id: 'an'
+              stability: experimental
+              value: 'AN'
+              brief: 'Antarctica'
+            - id: 'as'
+              stability: experimental
+              value: 'AS'
+              brief: 'Asia'
+            - id: 'eu'
+              stability: experimental
+              value: 'EU'
+              brief: 'Europe'
+            - id: 'na'
+              stability: experimental
+              value: 'NA'
+              brief: 'North America'
+            - id: 'oc'
+              stability: experimental
+              value: 'OC'
+              brief: 'Oceania'
+            - id: 'sa'
+              stability: experimental
+              value: 'SA'
+              brief: 'South America'
+      - id: continent.name
+        stability: experimental
+        type: string
+        brief: >
+          Name of the continent.
+        examples: [ 'North America', 'Europe' ]
+      - id: country.iso_code
+        stability: experimental
+        type: string
+        brief: >
+          Two-letter ISO Country Code ([ISO 3166-1 alpha2](https://en.wikipedia.org/wiki/ISO_3166-1#Codes)).
+        examples: [ 'CA' ]
+      - id: country.name
+        stability: experimental
+        type: string
+        brief: >
+          Country name.
+        examples: [ 'Canada' ]
+      - id: location.lon
+        stability: experimental
+        type: double
+        brief: >
+          Longitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ -73.614830 ]
+      - id: location.lat
+        stability: experimental
+        type: double
+        brief: >
+          Latitude of the geo location in [WGS84](https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84).
+        examples: [ 45.505918 ]
+      - id: name
+        stability: experimental
+        type: string
+        brief: >
+          User-defined description of a location.
+        note: >
+          User-defined description of a location, at the level of granularity they care about.
+          Could be the name of their data centers, the floor number, if this describes a local physical entity, city names.
```

Comment:
we have dedicated attributes for data center names. I assume there could be other attributes added in the future that could intersect with this one.

Depending on the signal it can be more suitable on  a span name, log payload field. There is also no instrumentation that's expected to add it.

Given all these potential issues, could we postpone adding this attribute until we have an actual use-case?

Reply:
https://github.com/open-telemetry/semantic-conventions/pull/1116#discussion_r1697244285